### ISSUE TITLE
Why decompress large zip files  is slower than decompress on PC?

### ISSUE THREAD

**Issue by wangjianpeng** [icsharpcode/SharpZipLib#159]
I use it in unity game , when decompress 270M file on PC ,just take at most 30s . but when i decompress on android , i takes  long time to decompress  ? does the sd card write and read will affect the performance ?

**Comment by LavenderMoon** [comment]
That and CPU speed and the number of threads you have available. I'd recommend having multiple archives and only loading them when needed.

**Comment by dpethes** [comment]
This isn't really an issue with SharpZipLib? It will be always faster on a decent desktop cpu vs. a mobile one.

**Comment by piksel** [comment]
Indeed, there is probably not much to do about it since it is mostly CPU-bound.
Storage performance might also play a role, but that's also outside the scope of this library.
